sdk/storage/storage-blob/src/highlevel.browser.ts

```diff
@@ -45,6 +48,36 @@ export async function uploadBrowserDataToBlockBlob(
   );
 }
 
+//type UploadBrowserDataToBlockBlobUrlOptions = IUploadToBlockBlobOptions & CredentialOptions & INewPipelineOptions;
+
+/**
+ * ONLY AVAILABLE IN BROWSERS.
+ *
+ * Uploads a browser Blob/File/ArrayBuffer/ArrayBufferView object to block blob given a url to that blob.
+ * This method assumes container already exists.
+ *
+ * When buffer length <= 256MB, this method will use 1 upload call to finish the upload.
+ * Otherwise, this method will call stageBlock to upload blocks, and finally call commitBlockList
+ * to commit the block list.
+ *
+ * @export
+ * @param {Blob | ArrayBuffer | ArrayBufferView} browserData Blob, File, ArrayBuffer or ArrayBufferView
+ * @param {BlockBlobURL} blockBlobURL
+ * @param {UploadBrowserDataToBlockBlobUrlOptions} [options] Upload browser data options & credential options & new pipeline options.
+ *                                                           If credential options is not specified {@link AnonymousCredential} is used.
+ * @returns {Promise<BlobUploadCommonResponse>}
+ */
+export async function uploadBrowserDataToBlockBlobUrl(
+  browserData: Blob | ArrayBuffer | ArrayBufferView,
+  url: string,
+  options: IUploadToBlockBlobOptions & CredentialOptions & INewPipelineOptions = {},
```

Comment:
I kept the intersection in the signature for the browser high level function.  It looks fine.  Also notice that `browserData` which has union type, looks fine too.

Reply:
options is out of sync with the jsdoc comment seems like